src/model/draft/mef-legato-services.yang

```diff
@@ -914,7 +921,9 @@ module mef-legato-services {
                          [MEF7.3] Section 7.4, 7.11";
                       leaf cos-name {
                         type leafref {
-                          path "../../../../../cos-names/cos-name/name";
+                          path "/mef-services/carrier-ethernet/" +
+	                       "subscriber-services/evc/cos-names/" +
+                               "cos-name/name";
                         }
```

Comment:
This is still broken; it is now refering to CoS Names in all EVCs instead of CoS names in this EVC.  Please revert to the relative path.  Same applies in all the cases below.  Please revert them all to relative paths.